notifier/selfstate/selfstate_test.go

```diff
@@ -154,6 +155,7 @@ func configureWorker(t *testing.T, isStart bool) *selfCheckWorkerMock {
 		LastCheckDelaySeconds:          120,
 		NoticeIntervalSeconds:          60,
 		LastRemoteCheckDelaySeconds:    120,
+		CheckInterval:                  1 * time.Second,
```

Comment:
I think one check per second might be to frequent
```suggestion
		CheckInterval:                  30 * time.Second,
```

Reply:
<img width="385" alt="image" src="https://github.com/moira-alert/moira/assets/15986169/27191102-4588-4ae1-975d-bbd7b087adc2">
i don't change test

Reply:
Ok